src/main/java/com/webdev/cosmo/cosmobackend/controller/VerificationController.java

```diff
@@ -0,0 +1,21 @@
+package com.webdev.cosmo.cosmobackend.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/verification")
+public class VerificationController {
+
+    @GetMapping("/unsecured")
+    public String getUnsecuredEndpoint(){
+        return "Unsecured";
```

Comment:
maybe change "secured" / "unsecured" into ResponseEntity being where we declare whether the option was ok or not?

Reply:
And it should be POST imho